hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/auth/HugeAuthenticator.java

```diff
@@ -64,6 +65,8 @@ public interface HugeAuthenticator extends Authenticator {
 
     UserWithRole authenticate(String username, String password, String token);
 
+    void unAuthenticate(AuthenticationFilter.Authorizer authorizer);
```

Comment:
sorry, can we change to this signature: `void unauthorize(SecurityContext context)`